main/api/src/mill/api/AggWrapper.scala

```diff
@@ -108,6 +108,7 @@ sealed class AggWrapper(strictUniqueness: Boolean){
       def isEmpty: Boolean = items.isEmpty
       def seq: scala.collection.TraversableOnce[V] = items
       def toTraversable: Traversable[V] = items.toTraversable
+      def iterator: Iterator[V] = items
```

Comment:
Does that mean, we should also extend `Iterable`?

Reply:
I think this change comes about because `TraversableOnce` has become an alias for `Iterable` in Scala 2.13, so we already extend it